### DISCUSSION TITLE
Vroid Hub SDKのサイトのデザインの崩れについて

### DISCUSSION THREAD

**Discussion by RearAlice** [pixiv/vroid-sdk-developers#68] [Q&A] [answered]
いつもお世話になっております。
久しぶりにサイト見ると、こんな感じになってるのですが、私だけの現象なのでしょうか？

MacのSafari
<img width="2240" alt="スクリーンショット 2023-03-14 14 12 31" src="https://user-images.githubusercontent.com/120567699/224901726-bd613489-b723-4e62-b91f-ca1576c9d898.png">

MS Edge&Google Chromeなどの場合
<img width="2240" alt="スクリーンショット 2023-03-14 14 12 51" src="https://user-images.githubusercontent.com/120567699/224901739-b82f0ab9-3d70-4490-9e25-53525f5eb5ec.png">

ちなみにSafariは、全履歴・キャッシュ削除してもこの状態です。
ご確認いただけると幸いです。

**Comment by nyamadan** [comment, marked as answer]
連絡ありがとうございます。

こちらでもレイアウト崩れを確認できましたので修正を進めます。

**Reply by nyamadan** [reply]
こちら、修正いたしました。

**Reply by RearAlice** [reply]
ありがとうございます。
